<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SimpleContactlistSettings</class>
 <widget class="QWidget" name="SimpleContactlistSettings">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>318</width>
    <height>238</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item row="9" column="0">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="4" column="0">
    <widget class="QCheckBox" name="extendedInfoBox">
     <property name="text">
      <string>Show extended icons</string>
     </property>
    </widget>
   </item>
   <item row="5" column="0">
    <widget class="QCheckBox" name="avatarsBox">
     <property name="text">
      <string>Show avatars</string>
     </property>
    </widget>
   </item>
   <item row="7" column="0">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Status icons size:</string>
     </property>
    </widget>
   </item>
   <item row="8" column="0">
    <widget class="QComboBox" name="sizesBox"/>
   </item>
   <item row="6" column="0">
    <widget class="QCheckBox" name="statusBox">
     <property name="text">
      <string>Show status text</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
